src/Content/Changes.php

```diff
@@ -62,6 +69,18 @@ public function files(): Files
 		return $files;
 	}
 
+	/**
+	 * Rebuilds the cache by finding all models with changes version
+	 */
+	public function generateCache(): void
+	{
+		foreach ($this->kirby->models() as $model) {
+			if ($model->version(VersionId::changes())->exists('*') === true) {
```

Comment:
`Question`: btw do we prefer this that much over writing `$model->version('changes')`?

Reply:
I'll leave it for now as it's the style used mostly at the moment. But maybe something to revisit (but then consistently everywhere).